### PR TITLE
Fix issue #107 #136 

### PR DESCRIPTION
**Pull request by HolmesJJ**
Update nice UI

### CODE REVIEW

File: src/main/java/seedu/address/model/ModelManager.java [hunk 1]
```diff
@@ -237,142 +233,6 @@ public void setMember(Member target, Member editedMember) {
...
-     */
-    @Override
-    public double getTotalAmountOfTransactionsPastSixMonth() {
-        // todo
-        return Math.random() + 6000;
-    }
-
     //=========== Filtered Member List Accessors =============================================================
 
     /**
```
Comment (zzybluebell) [inline src/main/java/seedu/address/model/ModelManager.java:238]: Great

File: src/main/java/seedu/address/ui/MemberDetailsCard.java [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
 import javafx.scene.control.Label;
-import javafx.scene.layout.FlowPane;
 import javafx.scene.layout.HBox;
 import javafx.scene.layout.Region;
+import javafx.scene.layout.StackPane;
 import seedu.address.commons.util.DateTimeUtil;
 import seedu.address.model.member.Member;
 import seedu.address.model.member.Tier;
+import seedu.address.model.reservation.Reservation;
+import seedu.address.model.transaction.Transaction;
```
Comment (zzybluebell) [inline src/main/java/seedu/address/ui/MemberDetailsCard.java:14]: Awesome

File: src/main/java/seedu/address/ui/MemberDetailsListPanel.java [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+            if (empty || member == null) {
+                setGraphic(null);
+                setText(null);
+            } else {
+                setGraphic(new MemberDetailsCard(member).getRoot());
+            }
+        }
+    }
+
+}
```
Comment (zzybluebell) [inline src/main/java/seedu/address/ui/MemberDetailsListPanel.java:45]: LGTM

File: src/main/resources/view/DarkTheme.css [hunk 4]
```diff
@@ -56,7 +56,11 @@
...
     -fx-border-width: 0 0 1 0;
     -fx-background-color: transparent;
-    -fx-border-color: transparent transparent derive(-fx-base, 80%) transparent;
+    -fx-border-color:
+        transparent
+        transparent
+        derive(-fx-base, 80%)
+        transparent;
     -fx-border-insets: 0 10 1 0;
 }
```
Comment (Morrow1ndy) [inline src/main/resources/view/DarkTheme.css:65]: Nice change here. Make UI prettier.

File: src/main/resources/view/MemberDetailsCard.fxml [hunk 5]
```diff
@@ -2,55 +2,79 @@
...
+      </HBox>
+      <HBox alignment="CENTER_LEFT">
+        <Label styleClass="cell_small_label, cell_label_width_100" text="Credit: " />
+        <Label fx:id="credit" styleClass="cell_small_label" text="\$credit" />
+      </HBox>
+      <HBox alignment="CENTER_LEFT">
+        <Label styleClass="cell_small_label, cell_label_width_100" text="Point: " />
+        <Label fx:id="point" styleClass="cell_small_label" text="\$point" />
+      </HBox>
+    </VBox>
```
Comment (Morrow1ndy) [inline src/main/resources/view/MemberDetailsCard.fxml:48]: Can modify the width to make it look nicer.

File: src/main/resources/view/MemberListCard.fxml [hunk 6]
```diff
@@ -28,27 +28,27 @@
         <Label fx:id="tier" text="\$tier" styleClass="cell_big_label" />
       </HBox>
       <HBox alignment="CENTER_LEFT">
-        <Label styleClass="cell_small_label, cell_small_label_width" text="ID: " />
+        <Label styleClass="cell_small_label, cell_label_width_80" text="ID: " />
```
Comment (Morrow1ndy) [inline src/main/resources/view/MemberListCard.fxml:31]: Good choice of width.

**Review by Morrow1ndy — COMMENTED** [review]
LGTM!

**Pull request merged**